### PR TITLE
Add a closurecompiler task to compile apprtc JS into one file

### PR DESCRIPTION
**Pull request by jiayliu**
No optimization is turned on for now, only whitespace and comments removal.

Change to the GAE push process and local deployment:
Must run grunt first, to generate the compiled js in samples/web/content/apprtc/js/compiled.

See http://gruntjs.com/getting-started for how to install grunt.

CC: @samdutton @marwahvikas @phoglund @KaptenJansson @chuckhays @tkchin

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -131,12 +155,14 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-jshint');
   grunt.loadNpmTasks('grunt-shell');
   grunt.loadNpmTasks('grunt-jstestdriver-phantomjs');
+  grunt.loadNpmTasks('grunt-closurecompiler');
```
Comment (chuckhays) [inline Gruntfile.js]: Should there be a matching change in package.json to specify installation of this package?

File: Gruntfile.js [hunk 2]
```diff
@@ -131,12 +155,14 @@ module.exports = function(grunt) {
...
   grunt.loadTasks('grunt-chrome-build');
 
   // set default tasks to run when grunt is called without parameters
   grunt.registerTask('default', ['csslint', 'htmlhint', 'jscs', 'jshint',
-                     'shell:runPythonTests', 'jstdPhantom']);
+                     'shell:runPythonTests', 'jstdPhantom',
+                     'closurecompiler:debug']);
   grunt.registerTask('build', ['grunt-chrome-build']);
```
Comment (chuckhays) [inline Gruntfile.js]: This step should probably be added to the build step as well. I'm happy to do it after this is in if you don't end up making any other changes.

File: Gruntfile.js [hunk 3]
```diff
@@ -120,8 +122,30 @@ module.exports = function(grunt) {
...
       },
       files: [
-        "samples/web/content/apprtc/js_test_driver.conf",
+        'samples/web/content/apprtc/js_test_driver.conf',
       ]},
+
+    closurecompiler: {
+      debug: {
+        files: {
+          'samples/web/content/apprtc/js/compiled/apprtc.debug.js': [
```
Comment (juberti) [inline Gruntfile.js]: fix indent

**Comment by juberti** [comment]
overall lgtm, aside from comment and build failure. Chuck can give final LGTM.

File: Gruntfile.js (hunk 3 above)
Comment (jiayliu) [inline Gruntfile.js]: eh, this is the right indent. The first file is the key.

File: Gruntfile.js (hunk 1 above)
Comment (jiayliu) [inline Gruntfile.js]: Done. PTAL

**Comment by chuckhays** [comment]
lgtm, I may have caused a merge conflict for you with my jscs cleanup.

**Pull request merged**